org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -264,6 +264,51 @@ GithubOidcCfnTemplateDeploy:
     Account: !Ref AdminCentralAccount
     Region: us-east-1
 
+# GH OIDC for lambda template deployments to AWS AdminCentral account
+GithubOidcLambdaTemplateDeploy:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.6/templates/IAM/github-oidc-provider.j2
+  StackName: !Sub ${resourcePrefix}-${appName}-lambda-template-deploy
+  Parameters:
+    ProviderArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-ProviderArn' ]
+    ProviderRoleName: !Sub ${resourcePrefix}-${appName}-lambda-template-deploy
+    PolicyDocument: |
```

Comment:
We currently use the [CfnDeployerServiceManagedPolicy](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/sceptre/admincentral/templates/iam-policies.yaml) which is used in [service-accounts.yaml](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/sceptre/admincentral/templates/service-accounts.yaml#L20). it might be appropriate to have more access but not less than that one, please double check.

Reply:
Ah, thanks, I've added a couple missing permissions

Reply:
please note for IT-3013 that we will need to remove the existing role after migrating to github actions.